### PR TITLE
grunt task to pull translations from GlotPress

### PR DESCRIPTION
**Pull request by zbtirrell**
See: https://central.tri.be/issues/36860

### CODE REVIEW

File: dev/grunt_options/glotpress_download.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+module.exports = {
+  core: {
+    options: {
+      url: 'http://translations.tri.be',
+      domainPath: 'lang',
+      slug: 'the-events-calendar',
+      textdomain: 'tribe-events-calendar',
+    },
+  },
```
Comment (borkweb) [inline dev/grunt_options/glotpress_download.js]: In JS, remove trailing commas on the last item in a list.

Comment (zbtirrell) [inline dev/grunt_options/glotpress_download.js]: Aren't trailing commas fine in the context of grunt since we are guaranteed to have modern ecmascript?

Comment (borkweb) [inline dev/grunt_options/glotpress_download.js]: Yeah, that's right. Even though trailing commas aren't in the JSON spec, doesn't mean we can't use them in config files. Keep 'em!

**Comment by borkweb** [comment]
:+1: Merge it

**Comment by borkweb** [comment]
Needs a URL update now

**Comment by borkweb** [comment]
:fallen_leaf:

**Pull request merged**